packages/toolpad-app/src/components/DialogForm.tsx

```diff
@@ -4,5 +4,4 @@ export default styled('form')({
   overflowY: 'auto',
   display: 'flex',
   flexDirection: 'column',
-  width: 400,
```

Comment:
@apedroferreira I'm reverting this, I don't think it's the right place to define dialog width, it's problematic when fullWidth is used on the dialog that contains this form. The component is intended to be reusable regardless of width of the form (it's in the components folder). Instead we should control dialog width using `maxWidth`  and `fullWidth` on the Dialog component

Reply:
ok, i remember trying other things before but for some reason there was a lot of shifting in the dialog
might just be unfamiliarity with MUI components
feel free to remove it